Client/mods/deathmatch/logic/luadefs/CLuaEngineDefs.cpp

```diff
@@ -186,10 +189,10 @@ void CLuaEngineDefs::AddClass(lua_State* luaVM)
         lua_classfunction(luaVM, "restoreMemorySize", "engineStreamingRestoreMemorySize");
         lua_classfunction(luaVM, "getBufferSize", "engineStreamingGetBufferSize");
         lua_classfunction(luaVM, "setBufferSize", "engineStreamingSetBufferSize");
-        lua_classfunction(luaVM, "setModelCacheLimits", "engineStreamingSetModelCacheLimits");
+        lua_classfunction(luaVM, "restoreBufferSize", "engineStreamingSetBufferSize");
```

Comment:
Why did you remove `setModelCacheLimits`?
Also, shouldn't you use `"engineStreamingRestoreBufferSize"` here?

Reply:
Um, perhaps I forgot to pull master.
I'll fix it.